pkg/common/constants/constants.go

```diff
@@ -65,8 +66,7 @@ const DaemonSetType = "DaemonSet"
 const PlaceholderContainerImage = "registry.k8s.io/pause:3.7"
 const PlaceholderContainerName = "pause"
 const PlaceholderPodRestartPolicy = "Never"
-const LabelPlaceholderFlag = "placeholder"
-const AnnotationPlaceholderFlag = DomainYuniKorn + "placeholder"
+const AnnotationPlaceholderFlag = DomainYuniKornInternal + "placeholder"
```

Comment:
quoted from [Craig's comment in jira](https://issues.apache.org/jira/browse/YUNIKORN-1351?focusedCommentId=17821767&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-17821767)

> Placeholder flag – used internally by scheduler only, remove documentation
> - internal.yunikorn.apache.org/placeholder [annotation] – generate in 1.6.0+
> - yunikorn.apache.org/placeholder [annotation] – parse in 1.6.0, remove in 1.7.0
> - placeholder [label] – parse in 1.6.0, remove in 1.7.0

Not sure if I misunderstand anything, but looks like we remove annotation `yunikorn.apache.org/placeholder` in current version (1.6.0), shouldn't we remove it in next version 1.7.0 ?

Reply:
Hi @brandboat, thanks for the heads up.
I think you're right. I miss it. Initially, I thought I could remove old placeholder label/annotation because it was used internally. 

After rethink about it, even though it's only be used by YuniKorn, I should still support old placeholder label/annotation in 1.6.0.
There might be some scenarios where old/new placeholder pod coexist after change version to 1.6.0.

Let me update this PR.

Reply:
Done.